pkg/apis/cnsoperator/config/cnsregistervolume_crd.yaml

```diff
@@ -10,62 +15,79 @@ spec:
-        status:
-          type: object
-          description: CnsRegisterVolumeStatus defines the observed state of CnsRegisterVolume
-          properties:
-            registered:
-              description: Indicates the volume is successfully imported.
-              type: boolean
-            error:
-              description: The last error encountered during import operation, if any.
-              type: string
-  version: v1alpha1
   versions:
   - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: CnsRegisterVolume is the Schema for the cnsregistervolumes API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: CnsRegisterVolumeSpec defines the desired state of CnsRegisterVolume
+            properties:
+              accessMode:
+                description: AccessMode contains the actual access mode the volume
+                  backing the CnsRegisterVolume has. AccessMode must be specified
+                  if VolumeID is specified.
+                type: string
+              diskURLPath:
+                description: 'DiskUrlPath is URL path to an existing block volume
+                  to be imported into Project Pacific cluster. VolumeID and DiskUrlPath
+                  cannot be specified together. DiskUrlPath is explicitly used for
+                  block volumes. AccessMode need not be specified and will be defaulted
+                  to "ReadWriteOnce". This field must be in the following format:
+                  Format: https://<vc_ip>/folder/<vm_vmdk_path>?dcPath=<datacenterName>&dsName=<datastoreName>
+                  Ex: https://10.192.255.221/folder/34a9c05d-5f03-e254-e692-02004479cb91/        vm2_1.vmdk?dcPath=Datacenter-1&dsName=vsanDatastore
+                  This is for a 34a9c05d-5f03-e254-e692-02004479cb91/vm2_1.vmdk file
+                  under datacenter "Datacenter-1" and datastore "vsanDatastore".'
+                type: string
+              pvcName:
+                description: Name of the PVC
+                type: string
+              volumeID:
+                description: VolumeID indicates an existing vsphere volume to be imported
```

Comment:
We have added some regex patterns for each string in the original spec. Can you please look and add them back